Core/src/main/java/org/tribuo/impl/ArrayExample.java

```diff
@@ -530,32 +530,42 @@ public boolean isDense(FeatureMap fMap) {
 
     @Override
     public void densify(List<String> featureList) {
```

Comment:
This should document that it expects a sorted list

Reply:
Fixed.